services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -6265,8 +6265,9 @@ public Long getLastSucceededExecutionId(String clusterName, String storeName) {
     if (storeName == null) {
       return getLastSucceedExecutionId(clusterName);
     } else {
+      String userStoreName = VeniceSystemStoreType.extractUserStoreName(storeName);
       return adminConsumerServices.containsKey(clusterName)
-          ? adminConsumerServices.get(clusterName).getLastSucceededExecutionId(storeName)
```

Comment:
Good catch! but depending on the deployment of this fix we may or may not get the execution ID ?

Reply:
This code path is only used by the parent controller when pending for local verification of the admin message is written successfully. I don't think deployment order matters here but feel free to give a concrete example if I missed something.

Reply:
I see this is used only in-memory map query in parent controller, so it should be fine.